frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -115,10 +124,35 @@ export default class FirebaseInteractor {
         name: userData.name as string,
         accountType: userData.accountType as AccountType,
         age: userData.age as number,
+        sessionId: userData.sessionId || -1,
+        onAssessment:
+          userData.onAssessment === undefined ? true : userData.onAssessment,
+        currentInterventionOrAssessment:
+          userData.currentInterventionOrAssessment || "oiBN8aE5tqEFK2gXJUpl",
       };
     }
   }
 
+  async updateCurrentUser(user: Partial<User>) {
+    this.db.collection("users").doc(this.auth.currentUser?.uid).update(user);
+  }
+
+  async createInitialAssessment() {
+    let words = await this.db.collection("words").get();
+    let wordIds = words.docs.map((word) => word.id);
+    let newAssessmentFields = {
+      currentIndex: 0,
+      // do we need to include in the beginning?
```

Comment:
Where would this be used?

Reply:
Sorry if I wasn't clear- I was responding to the comment in the code. Since we use increment for `durationInSeconds` and increment sets a value if the field does not exist, then a default value of 0 isn't needed when the assessment is created,  but specifying it at assessment creation time wouldn't hurt because it doesn't affect the value and makes it more clear in the code that `durationInSeconds` is an assessment field if that makes sense.

Reply:
Ah gotcha makes sense. I will keep it for clarity

Reply:
Sounds good! Also can you remove or update the comment?

Reply:
I will remove it